<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1039</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1039</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_classifying">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>165</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>45</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 16px;
}</string>
   </property>
   <property name="text">
    <string>CLASSIFYING</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">group_btns</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_learning">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>265</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>45</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 16px;
}</string>
   </property>
   <property name="text">
    <string>TRAINING</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">group_btns</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_testing">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>365</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>45</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 16px;
}</string>
   </property>
   <property name="text">
    <string>TESTING</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">group_btns</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_setup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>40</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 16px;
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 16px;
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">group_btns</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	alternate-background-color: rgba(0, 0, 0, 0);
	background-color: rgba(0, 0, 0, 0);
	color: rgb(227, 226, 228);
}</string>
   </property>
   <property name="text">
    <string>MAIN MENU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="group_btns"/>
 </buttongroups>
</ui>
